cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -561,10 +561,19 @@ private void printFoundArtifacts(ArrayList<GenerableArtifact> possibleArtifacts,
         Map<? super GenerableArtifact, Double> scores = new HashMap<>();
 
         for (int i = 0; i < matching.size(); i++) {
-            String description = isIncrements ? ((IncrementTo) matching.get(i)).getDescription()
-                : ((TemplateTo) matching.get(i)).getId();
-            JaccardDistance distance = new JaccardDistance();
-            scores.put(matching.get(i), distance.apply(description.toUpperCase(), userInput.toUpperCase()));
+        	if(!isIncrements) {
+        		String description = ((TemplateTo) matching.get(i)).getId();
+        		JaccardDistance distance = new JaccardDistance();
+                scores.put(matching.get(i), distance.apply(description.toUpperCase(), userInput.toUpperCase()));
```

Comment:
just a small formatting issue, not sure why, please check your eclipse editor, that you have run update-all-workspaces to get the correct formattings.

Reply:
Done.